### PR TITLE
Update teams-for-vdi.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
proposed changes

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -380,7 +380,7 @@ This feature is available in Teams version 1.3.00.13565 and later.
 ### Calling and meetings
 
 The following calling and meeting features are not supported:
-
+-Any multi window functionality like the new meeting experiences or any funcitionality that comes with the new meeting experience.
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md:383]:
```suggestion

- Any multi-window functionality like the new meeting experiences or any functionality that comes with the new meeting experience
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**